docs/src/main/asciidoc/podman.adoc

```diff
@@ -0,0 +1,58 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Using Podman with Quarkus
+
+https://podman.io/[Podman] is a daemonless container engine for developing, 
+managing, and running OCI Containers on your Linux System or other OS. 
+
+Podman could be used the same way as Docker with `podman-docker` package. The big andwantage is that you could run Podman as regular Linux user. 
+
+
+== Installing Podman
+
+The Podman package is available in several Linux distributions. To install it for your OS, please refer to https://podman.io/getting-started/installation[Podman install guide]. Below is the short instruction for popular Linux distributions. To install, run following commans.
+
+.Fedora
+
+[source,bash]
+----
+sudo dnf install podman podman-docker
+----
+
+.Ubuntu (21.04 and later)
+
+[source,bash]
+----
+sudo apt install podman podman-docker
+----
+
+=== After installation
+
+Podman is a daemonless container engine. Quarkus Dev Services and Testcontainers expect a running Docker daemon listening at a Unix socket. That's why following steps are required.
+
+[source,bash]
+----
+# Enable the podman socket with Docker REST API
+systemctl --user enable podman.socket --now
+# Set the required envvars
+export DOCKER_HOST=unix:///run/user/${UID}/podman/podman.sock
+export TESTCONTAINERS_RYUK_DISABLED=true
+----
+
+For detailed explanation, see the
+https://quarkus.io/blog/quarkus-devservices-testcontainers-podman/[blog article].
+
+To make changes permanent, add the exported environment variables to `~/.bashrc` file:
+[source,bash]
+----
+echo "export DOCKER_HOST=unix:///run/user/${UID}/podman/podman.sock" >> ~/.bashrc
+echo "export TESTCONTAINERS_RYUK_DISABLED=true" >> ~/.bashrc
+----
+
+=== Short names of images
+Testcontainers and Quarkus Dev Services also expect the container service they make requests against to be non-interactive. In case you have multiple registries configured in your Docker or Podman configuration, Podman responds with a prompt asking which registry should be used to pull containers from in case the containers pulled are specified by short name.
```

Comment:
```suggestion
Testcontainers and Quarkus Dev Services also expect the container service they make requests against to be non-interactive. In case you have multiple registries are configured in your Docker or Podman configuration, and when using short image names, Podman responds with a prompt asking which registry should be used to pull the images.
```